hypothesis-python/src/hypothesis/internal/conjecture/data.py

```diff
@@ -1935,14 +1935,6 @@ def permitted(f):
 }
 
 
-# eventually we'll want to expose this publicly, but for now it lives as psuedo-internal.
-def realize(value: object) -> object:
-    from hypothesis.control import current_build_context
-
-    context = current_build_context()
-    return context.data.provider.realize(value)
-
-
```

Comment:
hmm, I was imagining we'd use this helper in more places going forward to make tests support realizing backends...

Reply:
It can live in `tests.common.utils` then!  Mostly moving because I'd rather not ship stuff that I don't want users to touch unless we need to for some reason.

Reply:
very fair!